viz_scripts/scaffolding.py

```diff
@@ -92,6 +95,105 @@ def get_quality_text_ebike(all_confirmed_df, ebike_ct_df):
     print(quality_text)
     return quality_text
 
+def store_alt_text_generic(chart_description, chart_name, var_name, alt_text_file):
+    """ Inputs:
+    chart_description = what type of chart is it
+    chart_name = what to label chart by in the dictionary
+    var_name = the variable being analyzed across pie slices
+    alt_text_file = the alt text json file to save or update
+    """
+    # Fill out the alt text based on components of the chart and passed data
+    alt_text = f"{chart_description} chart of {var_name}."
+
+    # Make new dict if does not exist, store the new value
+    if os.path.exists(alt_text_file):
+        with open(alt_text_file, 'r') as f:
+            alt_text_dict = json.load(f)
```

Comment:
why do you need to load the existing `alt_text` first? we regenerate the plot every time, not incrementally. So we will have the full data on each run...

Reply:
I was loading the alt_text_lookup since each notebook/function call will be writing to it, and I didn't want it to get overwritten and wiped every time. I could try opening the file with 'a' and append to the json, but this seemed logical.

Reply:
why not just create a separate file per plot, similar to the figure? That will also make the website code easier because you don't need to use javascript to read and parse the file. Ideally, there would be a way to include the alt-text directly from the file name but I'm not sure there's an easy way to do that.

Reply:
The closest approach that I can think of to load the file directly is
http://stackoverflow.com/questions/15320801/ddg#15320839
but not sure it works with attributes

Reply:
Modified to write separate .html files for each plot, not sure of format for now but can work it out when I make the frontend changes.